### ISSUE TITLE
Towards Yolov5

### ISSUE THREAD

**Issue by samyhocine** [ifzhang/FairMOT#139]
Hi,

Yolov4 and v5 are out now and their benchmark are outstanding compared to Yolov3.

Are you going to update FairMOT with the last version of Yolo ?
Does it mean retraining all your model ?

Best regards,

Samy

**Comment by ifzhang** [comment]
Yes, that will be totally different network structure and needs to be trained again. I have no plans to update it now because it is really a hard work~

**Comment by CaptainEven** [comment]
@samyhocine You can refer to my recent work:
 https://github.com/CaptainEven/YOLOV4_MCMOT, which is an extention work of FairMOT.
